### PR TITLE
Code formatting fixes

### PR DESCRIPTION
**Pull request by AttemptToCallNil**
Looks like I have the workflow more or less set up. This probably isn't that useful, but not that useless either.

### CODE REVIEW

File: src/background.js [hunk 1]
```diff
@@ -10,17 +10,17 @@ import { getWikiInfo } from "./libraries/wikiDetect.js";
...
-//Should hook into node project varaible
+// Should hook into node project varaible
 const projectName = "MediaWikiAGE";
 
 const isDevelopment = process.env.NODE_ENV !== "production";
 
 const isMac = process.platform !== "darwin";
 
-//Load Config
+// Load Config
```
Comment (Sidemen19) [inline src/background.js:23]: Weird capitalisation throughout the sentence

File: src/background.js [hunk 2]
```diff
@@ -37,29 +37,29 @@ const spellbook = {
     this.settings.farms[add.key] = { ...(this.settings.farms[add.key] || {}), ...add.val };
   },
   loadSettings: function() {
-    //Load Settings
+    // Load Settings
```
Comment (Sidemen19) [inline src/background.js:40]: Ditto

File: src/background.js [hunk 3]
```diff
@@ -97,7 +97,7 @@ const spellbook = {
     });
   },
   addSingleUser: function(username, password, url, note) {
-    /*Probable structure
+    /* Probable structure
```
Comment (Sidemen19) [inline src/background.js:100]: Could probably add another space to the end of this block comment

File: src/background.js [hunk 4]
```diff
@@ -116,11 +116,11 @@ const spellbook = {
       throw new ErrInput("Password can't be undefined");
     const scriptPath = new URL(url);
 
-    //Chunk Load Script Path
+    // Chunk Load Script Path
```
Comment (Sidemen19) [inline src/background.js:121]: Weird capitalisation again

File: src/background.js [hunk 5]
```diff
@@ -228,13 +228,13 @@ const spellbook = {
 };
 spellbook.loadSettings();
 
-//Hidden Testing
+// Hidden Testing
```
Comment (Sidemen19) [inline src/background.js:236]: Ditto

File: src/background.js [hunk 6]
```diff
@@ -266,7 +266,7 @@ async function createWindow() {
   if (process.env.WEBPACK_DEV_SERVER_URL) {
     // Load the url of the dev server if in development mode
     await win.loadURL(process.env.WEBPACK_DEV_SERVER_URL);
-    //if (!process.env.IS_TEST) win.webContents.openDevTools();
+    // if (!process.env.IS_TEST) win.webContents.openDevTools();
```
Comment (Sidemen19) [inline src/background.js:274]: May as well remove this comment, no point in having it around

**Review by Sidemen19 — APPROVED** [review]

File: src/background.js (hunk 6 above)
Comment (AttemptToCallNil) [reply src/background.js]: Removed.

File: src/background.js (hunk 5 above)
Comment (AttemptToCallNil) [reply src/background.js]: Fixed.

File: src/background.js (hunk 4 above)
Comment (AttemptToCallNil) [reply src/background.js]: Fixed.

File: src/background.js (hunk 3 above)
Comment (AttemptToCallNil) [reply src/background.js:100]: Done. I think.

File: src/background.js (hunk 2 above)
Comment (AttemptToCallNil) [reply src/background.js]: Fixed.

File: src/background.js (hunk 1 above)
Comment (AttemptToCallNil) [reply src/background.js]: Fixed.

Comment (Helcostr) [reply src/background.js]: Capitalization was styled after "Titles, and not sentences XD"

whoops

**Pull request merged**